docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -23,6 +23,16 @@ Install 4.y (The new version)
 
 Replace “y” with the new minor and patch release number, for example, 4.0.0.zip
 
+
+.. note::
+    For versions 4.7.* and above, use the format `4.7.*.zip`, for example:
+
+    .. code-block:: bash
+
+        wget -L https://github.com/scylladb/scylla-monitoring/archive/4.7.1.zip
```

Comment:
you can change the wget command to use the -O tag, so it will work regardless of the version number

Reply:
I'm not sure I got the point here - with `-O` we can specify a file-name-target but the URL-suffix does matter for this case, `scylla-monitoring-x.y.z` vs `x.y.z`,

Reply:
it's not a huge change, just make the second line simpler, I wander if we can use the same parameters for the latest version.